File: bin/Practica2Python/Genero.py
class Genero:
    # Atributos
    nombre=""
    edad_recomentada=0
    #Método constructor
    def __init__(self,nombre,edad_recomentada):
        self.nombre=nombre
        self.edad_recomentada=edad_recomentada
    # Getters y setters
    def getNombre(self):
        return self.nombre
    def setNombre(self,pNombre):
        self.nombre=pNombre
    def getEdad_recomendada(self):
        return self.edad_recomentada
    def setEdad_recomendada(self,pEdad_recomendada):
        self.edad_recomentada=pEdad_recomendada

File: src/Practica2/Genero.java
package Practica2;

public class Genero {
    /*
    Aquí tenéis que poner los siguientes atributos
     * nombre: El nombre del género.
     * edad_recomendada: La edad mínima recomendada.
     */
    private String nombre;
    private int edad_recomendada;

    
    
    //Método constructor
    public Genero(String pNombre,int pEdad_recomendada){
        this.nombre=pNombre;
        this.edad_recomendada=pEdad_recomendada;
    }

    // Getters y setters
    public String getNombre(){
        return this.nombre;/*se puede poner return nombre; */
    }
    public void setNombre(String pNombre){
        this.nombre=pNombre;
    }
    public int getedad_recomendada(){
        return this.edad_recomendada;
    }
    public void setedad_recomendada(int pedad_recomendada){
        this.edad_recomendada=pedad_recomendada;
    }

        

}
